python/pdo/test/contract.py

```diff
@@ -414,7 +463,7 @@ def LocalMain(config) :
         ErrorShutdown()
 
     try :
-        UpdateTheContract(config, enclave, contract, contract_creator_keys)
+        UpdateTheContract(config, contract, enclaves, contract_creator_keys)
     except Exception as e :
```

Comment:
not sure i understand why you changed the order of the parameters?

Reply:
thank you, reverted back, (in the first implementation there were no enclaves, but then added back after our last discussion, but changed the order, have fixed now)